spec/event_stores_spec.rb

```diff
@@ -77,5 +84,37 @@ module Sandthorn
         expect(stores[:foo]).to eq(store)
       end
     end
+
+    describe "#map_aggregate_type_to_event_store" do
+
+      let(:klass) {
+        class AnAggregate 
+          include Sandthorn::AggregateRoot
+        end
+      }
+
+      it "maps the aggregate_type to the event_store" do
+        store = double
+        stores.add(:foo, store)
+        stores.map_types(foo: [klass])
+        expect(klass.event_store).to eq(:foo)
+      end
+    end
+
+    describe "#map_aggregate_types_to_event_store" do
+
+      let(:klass) {
+        class AnAggregate 
```

Comment:
This "should not work", i.e. we should get a error saying the constant is already defined, but I'm guessing rspec does some random magic to make it work. But I still think we should prefer anonymous classes over this (`klass = Class.new(Foo)`.